### PR TITLE
Noted that journald should be used for aggegated logging

### PR DESCRIPTION
**Pull request by ahardin-rh**
Addresses https://github.com/openshift/openshift-docs/issues/4185

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@jcantrill PTAL

File: install_config/aggregate_logging.adoc [hunk 1]
```diff
@@ -47,6 +47,9 @@ securely.
 xref:../install_config/install/host_preparation.adoc#managing-docker-container-logs[Managing
 Docker Container Logs] discusses the use of `json-file` logging driver options
 to manage container logs and prevent filling node disks.
+
+If deploying aggregated logging, it is strongly recommended that you
```
Comment (portante) [inline install_config/aggregate_logging.adoc]: Can we make this "stronger" still?

    "Aggregated logging is only supported using the `journald` driver in Docker."

Or something like that?

**Comment by ahardin-rh** [comment]
@portante This is updated. Thanks!

**Review by jcantrill — APPROVED** [review]

**Comment by ahardin-rh** [comment]
[rev_history]
|xref:../install_config/aggregate_logging.adoc#install-config-aggregate-logging[Aggregating Container Logs]
|Noted that aggregated logging is only supported using the `journald` driver in Docker
%
|xref:../ install_config/install/host_preparation.adoc#install-config-install-host-preparation[Installing a Cluster -> Host Preparation]
|Noted that aggregated logging is only supported using the `journald` driver in Docker
%

**Pull request merged**